src/legacy/status_im/events.cljs

```diff
@@ -124,7 +123,7 @@
       :content             (i18n/label :t/biometric-auth-confirm-message)
       :confirm-button-text (i18n/label :t/biometric-auth-confirm-try-again)
       :cancel-button-text  (i18n/label :t/biometric-auth-confirm-logout)
-      :on-accept           #(biometric/authenticate nil {:on-fail on-biometric-auth-fail})
+      :on-accept           #(rf/dispatch [:biometric/authenticate {:on-fail on-biometric-auth-fail}])
```

Comment:
I would still like to move this `on-biometric-auth-fail` out of "legacy" albeit perhaps we do in a separate issue?